ADA/databricks_notebooks.qmd

```diff
@@ -0,0 +1,54 @@
+---
+title: "Databricks Notebooks"
+---
+
+------------------------------------------------------------------------
+
+## Notebooks
+
+------------------------------------------------------------------------
+
+Notebooks are a special kind of script that Databricks supports. They consist of code blocks and markdown blocks which can contain formatted text, links and images. Due to the ability to combine markdown with code they are very well suited to creating and documenting data pipelines, such as creating a core dataset that underpins your other products. They are particularly powerful when parameterised and used in conjuction with [Workflows](ADA/Databricks_workflows.qmd).
+
+You can create a notebook in your workspace, either in a folder or a repository.\
+To do this locate the folder/repository you want to create the notebook in then click the 'Create' button and select Notebook.
+
+::: callout-tip
+It's generally recommended that any notebooks used for core business processes are created in a repository linked to GitHub/DevOps where they can be version controlled.
+:::
+
+Once you've created a notebook it will automatically be opened. Any changes you made are saved in real time. In order to 'save' a snapshot of your work it is recommended to use git commits.
+
+You can change the title from 'Untitled Notebook *\<timestamp\>*', and set it's default language in the drop down immediately to the right of the notebook title.
+
+![](/images/ada-notebook.png)
+
+The default language is the language the notebook will assume all code chunks are written in. In the screenshot above the default language is 'R', so all chunks will be assumed to be written in R unless otherwise specified.
+
+You can also add markdown cells to add text, links and graphics to your notebook in order to document the processing done within it.
+
+To add a new code of markdown chunk move the mouse above or below another chunk and the buttons '+Code' and '+Text' will appear.
+
+![](/images/ada-notebook-add-chunk.png)
+
+To run code chunks you'll first need to attach your compute resource to it by clicking the 'Connect' button in the top right hand side of the page.
+
+![](/images/ada-notebook-attach.png)
+
+You can run a code chunk either by pressing the play button in the top left corner of the chunk, or by pressing Ctrl + Return/Enter on the keyboard. Any outputs that result from the code will be displayed underneath the chunk.
```

Comment:
On the outputs - maybe a bit too niche for this page but I know View() doesn't work in Databricks for example, do we need to mention other ways for people to customise what output they see or how much of it they see?

Reply:
I've added a call out box to note that this is the case in R. Not sure how much more detail we can go into though as the display() function doesn't have any documentation so I'm not sure if there are ways to limit it other than specifying a limit to the data you're trying to show